src/robot/libraries/dialogs_ipy.py

```diff
@@ -12,32 +12,183 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import wpf
+from System.Windows import Application, GridLength, SizeToContent, TextWrapping, \
+                           Thickness, Window, WindowStartupLocation, WindowState
+from System.Windows.Controls import Button, ColumnDefinition, Grid, Label, ListBox, \
+                                    PasswordBox, RowDefinition, TextBlock, TextBox
```

Comment:
dialogs_jy.py used \, dialogs.py used () and pylint complained about the ()...

I actually think the parentheses look better.